integration-tests/smoke/vrfv2plus_test.go

```diff
@@ -1329,23 +1329,40 @@ func TestVRFV2PlusWithBHS(t *testing.T) {
 		randRequestBlockNumber := randomWordsRequestedEvent.Raw.BlockNumber
 		var wg sync.WaitGroup
 		wg.Add(1)
-		//Wait at least 256 blocks
-		_, err = actions.WaitForBlockNumberToBe(
-			randRequestBlockNumber+uint64(257),
-			sethClient,
-			&wg,
-			configCopy.VRFv2Plus.General.WaitFor256BlocksTimeout.Duration,
-			t,
-			l,
-		)
+
+		waitForNumberOfBlocks := 257
```

Comment:
Perhaps we could consider this number to be configurable in the future? Pectra upgrade will bring EIP-2935 which will store up to 8192 block hashes.

Reply:
so do you mean that when invoking blockhash method in Solidity, it will store last 8192 block hashes, instead of 256?